docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "test-bruno",
+  "type": "collection"
```

Comment:
Consider adding a description field to the JSON configuration to provide more context about its purpose. [medium]